source/sass/glyphs.scss

```diff
@@ -1,26 +1,34 @@
-@mixin glyph($id, $width: 20px, $height: 20px) {
+$defaultGlyphPath: "../_images/glyphs/";
```

Comment:
Use hyphen-case for SCSS variables. e.g., `$default-glyph-path`

Reply:
ah okay, I'll update that.